### DISCUSSION TITLE
How to show `defaultLocale` if other languages are not available?

### DISCUSSION THREAD

**Discussion by PSIget** [shuding/nextra#2054] [Q&A]
For example, I have 2 languages:
- en = defaultLocale
- uk

And I have a page `/about` and it only translates into the main language (en) but if the user with the language `uk` follows the link `/uk/about`, he will get a 404 error.

How can I avoid the 404 error and show the user the original language(en)? And, if possible, show him a callout with the text "This page has not yet been translated into your language"?

**Comment by danieljaguiar** [comment]
Did you get a solution for this?
